### DISCUSSION TITLE
How to delete skins from lazer?

### DISCUSSION THREAD

**Discussion by Sp33dW33d** [ppy/osu#18645] [Q&A] [answered]
Beatmaps, rulesets etc can be deleted but I can't find any way to delete my skins

**Comment by frenzibyte** [comment, marked as answer]
You can delete all skins in game settings. Deleting specific ones will be available in today's release (see https://github.com/ppy/osu/pull/18609).

**Reply by Sp33dW33d** [reply]
Yeah I meant individual skins actually. Kinda funny how it was implemented right after I made this post lol
